index.html

```diff
@@ -1,247 +1,380 @@
+                rows="10"
+                minlength="30"
+                placeholder="Enter text here"
+                required
+              ></textarea>
+            </label>
+          </div>
+          <button type="submit" clear>Get in touch</button>
+        </fieldset>
+      </form>
+
+      <form
+        action="https://formspree.io/f/xpznwejk"
+        class="form2"
+        method="POST"
+      >
+        <fieldset>
+          <div class="input-group">
+            <label>
+              <input
+                type="text"
+                id="fullName"
+                name="fullName"
+                placeholder="Full name"
+                required
+              />
+            </label>
+          </div>
+        
+          <div class="input-group">
+            <label>
+              <input
+                type="email"
+                id="email"
+                name="email"
+                placeholder="name@gmail.com"
+                required
+              />
+            </label>
+          </div>
+          <div class="input-group">
+            <label>
+              <textarea
+                name="msg"
+                id="msg"
+                cols="30"
+                rows="10"
+                minlength="30"
+                placeholder="Enter text here"
+                required
+              ></textarea>
+            </label>
+          </div>
+          <button type="submit">Get in touch</button>
+        </fieldset>
+      </form>
+    </footer>
+
+    <!-- <footer id="contact">
+      <div class="contact-text">
+        <p></p>
+      </div>
+      <form action="https://formspree.io/f/xpznwejk" class="form" method="POST">
+        <fieldset>
+          <div class="allNames">
+            <div class="input-group allNames1">
+            <label>
+            <input type="text" id="firstName" class="fName" name="firstName" placeholder="First name" required> 
+          </label>
+        </div>
+          <div class="input-group allNames1">
+            <label>
+            <input type="text" id="lastName" class="lName" name="lastName" placeholder="Last name" required>
+          </label>
+          </div>
+          </div>
+          <div class="input-group fullN">
+            <label>
+            <input type="text" id="fullName" name="fullName" placeholder="Full name" required>
+          </label>
+          </div>
+          <div class="input-group">
+            <label>
+            <input type="email" id="email" name="email" placeholder="name@gmail.com" required>
+          </label>
+          </div>
+          <div class="input-group">
+            <label>
+            <textarea name="msg" id="msg" cols="30" rows="10" minlength="30" placeholder="Enter text here" required></textarea>
+          </label>
+          </div>
+          <div class="submit">
+            <button type='submit'>Get in touch</button>
+          </div>
+        </fieldset>
+      </form>
+     </footer> -->

```

Comment:
- [ ] Dear @prg-04 , You did well implementing some of the suggested changes from previous code reviewers. Yet there is one main requirement you have missed at this point. I see you have commented some codes on your code base. This is against good coding practice. Also will make your codebase a bit messy. Remember you can always go back in history (using your commit history) to check any code you have committed once upon a time.